gatsby/content/projects/2020-01-04-native-gitter-bridge.mdx

```diff
@@ -35,4 +35,6 @@ If you're already using [Element](https://element.io), you can browse all of the
 ✗ Read Receipts  
 ✗ Typing notifications
 
-If you're curious about the bridge, there is a great overview of the capabilities and future plans in [this Matrix blog post](https://matrix.org/blog/2020/12/07/gitter-now-speaks-matrix).
+If you're curious about the bridge, there is a great overview of the capabilities and future plans in [this Matrix.org blog post](https://matrix.org/blog/2020/12/07/gitter-now-speaks-matrix).
+
+![](/docs/projects/images/gitter-bridge-demo.gif)
```

Comment:
I can't get this image to show up locally 🤷 

It's in the same directory as `/docs/projects/images/gitter-logo.svg` which works for the `thumbnail`. I also can't get any new `test.png` images to work either even in the `thumbnail` field so this feels like a stupid Gatsby cache problem. Already tried `npm run clean` for `gatsby clean`

Reply:
```suggestion
![](/docs/projects/bridges/images/gitter-bridge-demo.gif)
```

Reply:
🤦‍♂️ ugh, I guess we have `gitter-logo.svg` in both places which made this hard to notice

Thank you @afranke 🌟